### PR TITLE
KAIRScares 09-17-2020 

### PR DESCRIPTION
**Pull request by ruchipatel1**

### CODE REVIEW

**Review by stephanieluo18 — APPROVED** [review]
looks gucci

**Comment by nidhishad** [comment]
looks good!

File: foo [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello World! hello
```
Comment (Arthelon) [inline foo]: Please delete this file!

**Review by Arthelon — CHANGES_REQUESTED** [review]
Looks good! just delete that extraneous `foo` file

**Review by Arthelon — APPROVED** [review]

**Pull request merged**